lib/database_connection.rb

```diff
@@ -0,0 +1,16 @@
+require 'pg'
+
+class DatabaseConnection
+  def self.setup(dbname)
+    @connection = PG.connect(dbname: dbname)
+  end
+
+  def self.connection
+    @connection
+  end
+
+  def self.query(sql)
+    @connection.exec(sql)
+  end
+  
```

Comment:
amazing work in extracting the DatabaseConnection and related methods from the Peep class - very good example of SRP principle